src/components/FeatureViews/Scheduler/SchedulerUtils.js

```diff
@@ -34,4 +36,87 @@ export const BootstrapInput = withStyles(theme => ({
             boxShadow: '0 0 0 0.2rem rgba(0,123,255,.25)',
         },
     },
-}))(InputBase);
\ No newline at end of file
+}))(InputBase);
+
+export const handleToolTip = (info) => {
+    function formatDate(start, end) {
+        const MonthConverter = {
+            "0": "January",
+            "1": "February",
+            "2": "March",
+            "3": "April",
+            "4": "May",
+            "5": "June",
+            "6": "July",
+            "7": "August",
+            "8": "September",
+            "9": "October",
+            "10": "November",
+            "11": "December",
+        };
+
+        const date = new Date(start);
+        const dateNumber = date.getDate();
+        const dayOfWeek = date.getDay();
+        const startMonth = date.getMonth();
+        // Gets days
+        const Days = DayConverter[dayOfWeek];
+
+        // Gets months
+        const Month = MonthConverter[startMonth];
+
+        // Start times and end times variable
+        const startTime = new Date(start).toTimeString();
+        const endTime = new Date(end).toTimeString();
+        // Converts 24hr to 12 hr time
+        function timeConverter(time) {
```

Comment:
we should have a function for this in FormUtils.js and RegistrationCourse.js, check to see if they're being used. I don't think they are, so go ahead and delete them, but separate this function out to its own function.